stable/stackdriver-exporter/templates/deployment.yaml

```diff
@@ -23,6 +23,10 @@ spec:
       labels:
         app: {{ template "stackdriver-exporter.name" . }}
         release: "{{ .Release.Name }}"
+      annotations:
+      {{- range $key, $value := .Values.annotations }}
```

Comment:
it will be more readable using 

``` 
{{- if .Values.service.serviceAnnotations }}
 annotations:
 {{ toYaml .Values.service.serviceAnnotations | indent 4 }}
 {{- end }}

```